ansible/roles/kind/kind_cluster_create/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Render kind config
+  block:
+    - template:
+        src: kind.yaml.j2
+        dest: /tmp/kind.yaml
+    - set_fact:
+        kind_config: /tmp/kind.yaml
+  when: 'kind_config == ""'
+- name: Deploy Kind cluster
+  shell:
+    cmd: kind create cluster --name {{project}} --kubeconfig {{kubeconfig}} --config {{kind_config}}
```

Comment:
What will happen if a config file was passed to the playbook, and the config file contains the name field, what name will take precedence?

I think since the config file is always used (whether it was passed or rendered) it would be a good idea to add the `name` field to the jinja file so that all cluster configs are in one place (except the kubeconfig since it looks like it is not configurable using the config file).

Reply:
`--name` overrides the name in config file, it is intended to name the cluster with the ci project name